swagger-spec/subjects/subject_definition.yaml

```diff
@@ -0,0 +1,112 @@
+type: object
+title: Subject
+properties:
+  id:
+    type: string
+    readOnly: true
+    description: 'The unique identifier of the Subject.'
+  type:
+    type: string
+    readOnly: false
```

Comment:
should all subject properties be `readOnly: true`, or am i forgetting how they're editable?